### ISSUE TITLE
Empty scene

### ISSUE THREAD

**Issue by yasmeen2001234** [n3urovirtual/PicoXR_EyeTracking_Demo#1]
I opened the Sample_Project however i did not find  the objects and the scripts for eyetracking

**Comment by n3urovirtual** [comment]
Yes, that's true. As I mention in the description of the project "There is an upcoming update of the eye tracking API scheduled for March 2023. If no major changes are made and the project continues to function properly, the source code will be made available on this repository."

**Comment by Pirate2606** [comment]
Hi @n3urovirtual , is there any update for this project? I wanted to try the eye tracking feature of the PICO device.

**Comment by ranssr** [comment]
Hi @n3urovirtual , is there any update for this project? I am looking forward to testing your demo.Thanks a lot.

**Comment by XuJicang** [comment]
this project die?
